schema.graphql

```diff
@@ -130,3 +130,10 @@ type Global @entity {
   numOpenConditions: Int!
   numClosedConditions: Int!
 }
+
+type CuratedMarket @entity {
```

Comment:
Instead of creating a new entity, why not add the fields to the existing FixedProductMarketMaker. For example:

* `klerosTCRItemId`
* `klerosTCRStatus`
* `curatedByKleros`

This would help with filtering markets on the frontend, given the limited types of filters available through the Graph node interface.